ocp_resources/forkliftcontroller.py

```diff
@@ -0,0 +1,35 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ForkliftController(NamespacedResource):
+    """
+    Migration Toolkit For Virtualization (MTV) ForkliftController object
+    """
+
+    api_group = NamespacedResource.ApiGroup.FORKLIFT_KONVEYOR_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        data=None,
```

Comment:
Is this here by accident?

Reply:
Yes. Thanks.
Currently, we need nothing but to get the ForkliftController resource